meta.go

```diff
@@ -294,29 +307,31 @@ func (m Meta) GetYAML() (tag string, value interface{}) {
 		return mrs
 	}
 	return "", struct {
-		Name        string                       `yaml:"name"`
-		Summary     string                       `yaml:"summary"`
-		Description string                       `yaml:"description"`
-		Provides    map[string]marshaledRelation `yaml:"provides,omitempty"`
-		Requires    map[string]marshaledRelation `yaml:"requires,omitempty"`
-		Peers       map[string]marshaledRelation `yaml:"peers,omitempty"`
-		Categories  []string                     `yaml:"categories,omitempty"`
-		Tags        []string                     `yaml:"tags,omitempty"`
-		Subordinate bool                         `yaml:"subordinate,omitempty"`
-		Series      []string                     `yaml:"series,omitempty"`
-		Terms       []string                     `yaml:"terms,omitempty"`
+		Name           string                       `yaml:"name"`
+		Summary        string                       `yaml:"summary"`
+		Description    string                       `yaml:"description"`
+		Provides       map[string]marshaledRelation `yaml:"provides,omitempty"`
+		Requires       map[string]marshaledRelation `yaml:"requires,omitempty"`
+		Peers          map[string]marshaledRelation `yaml:"peers,omitempty"`
+		Categories     []string                     `yaml:"categories,omitempty"`
+		Tags           []string                     `yaml:"tags,omitempty"`
+		Subordinate    bool                         `yaml:"subordinate,omitempty"`
+		Series         []string                     `yaml:"series,omitempty"`
+		Terms          []string                     `yaml:"terms,omitempty"`
+		MinJujuVersion *version.Number              `yaml:"min-juju-version,omitempty"`
```

Comment:
not really clear why this is a pointer -- surely the zero version.Number is perfectly suitable, and less likely to induce potential panics when not initialized...